app/models/role.rb

```diff
@@ -171,6 +175,7 @@ def clone(role_params = {})
   end
 
   def locked?
+    return false if self.class.bypass_locking
```

Comment:
Why is this needed? the role is still locked hence it should return true, we're just bypassing it. If that's because of filters validations/filters, I think [existing check](https://github.com/dLobatog/foreman/blob/fc614898bea04d0998d01f4411fdb2e0357d39bf/app/models/filter.rb#L226) should be extended